open_schools_platform/organization_management/organizations/selectors.py

```diff
@@ -36,3 +39,11 @@ def get_organizations_by_user(user: User) -> QuerySet:
 
     return qs if len(qs) == 0 else \
         get_organizations(filters={"ids": ','.join(list(map(lambda x: str(x.organization.id), list(qs))))})
+
+
+def get_organization_circle_queries(organization: Organization):
+    org_circles = organization.circles.values()
+    queries = get_queries(filters={"recipient_ids": form_ids_string_from_queryset(org_circles),
+                                   "sender_ct": ContentType.objects.get(model="studentprofile")},
+                          empty_filters=True)
+    return queries
```

Comment:
should we locate it to services?

Reply:
I think it is better to leave it here because we are extracting data from db in this function.